### DISCUSSION TITLE
Interface filter in translation field

### DISCUSSION THREAD

**Discussion by Valdri** [directus/directus#21546] [Feature Requests]
### Summary

Interface filter in the translation field with the ability to filter the languages available for translation

### Motivation

I have a product with 2 translation fields containing the product description, one for amazon the other for woocommerce
translations_amazon
translations_shop

They use the languages data model as a list of languages

in languages I have 3 entries
de (German), cz (Czech), fr (French)

Now amazon has no Czech version and in the list of languages available for translation I would like to leave only de and fr

### Detailed Design

Interface filter in the translation field with the ability to filter the languages available for translation

### Requirements List

Interface filter in the translation field with the ability to filter the languages available for translation

### Drawbacks

I can't think of any drawback, it will be a new option in the interface

### Alternatives

duplication of the language data model for each translation field
languages_amazon -> translations_amazon
languages_shop -> translations_shop

### Adoption Strategy

This is not a breaking change

**Comment by m4rrc0** [comment]
Hey.
I have another (rather complex) use case for this.
Consider this simpler version:
- A large array of languages
- Users opt-in to manage some languages (could be on their user profile)
- Only user-selected languages are shown on the interface for the current user

(My actual use case has a few more layers: users are part of teams, teams manage projects, languages are associated to projects...)

With the m2m field it seems we are able to filter (and with m2o as well). As I understand, the translation field is based on a m2m relation so it seems reasonable to think filtering should not be an issue.

I am willing to explore creating an extension for this. I'd really appreciate some pointers from the team before diving in.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
